bands/automated_emails.py

```diff
@@ -12,22 +12,22 @@ def __init__(self, *args, **kwargs):
 BandEmail('{EVENT_NAME} Performer Checklist', 'band_notification.txt')
 
 BandEmail('Last chance to apply for a {EVENT_NAME} Panel', 'band_panel_reminder.txt',
-          lambda b: not b.completed_panel and days_before(3, c.BAND_PANEL_DEADLINE))
+          lambda b: not b.panel.completed and days_before(3, c.BAND_PANEL_DEADLINE))
```

Comment:
If you do it this way you'll have a stacktrace written out to the logs every time the automated email background task checks a band which doesn't have a ``.panel`` or a ``.info``, etc.  You should probably implement a ``.completed_panel`` property or something; otherwise you need to say ``(not b.panel or not b.panel.completed)`` here.